.env.example

```
@@ -15,6 +15,8 @@ TI_WEB_HOST=0.0.0.0
 TI_WEB_PORT=3000
 TI_WEB_USE_TLS=false
 TI_WEB_APP_STATIC_CACHE_DISABLED=false
+# Comma-separated admin allowlist; matched against the session user's ID, username or email.
+TI_WEB_AUTH_ADMINS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TI_WEB_AUTH_ADMINS key should go before the TI_WEB_HOST key

(UnorderedKey)